content/en/continuous_integration/tests/ruby.md

```diff
@@ -336,28 +332,37 @@ Datadog.configure do |c|
   # The name of the service or library under test
   c.service = 'my-ruby-app'
 
-  # Enables the Cucumber instrumentation
-  c.ci.instrument :cucumber
+  # Enables the RSpec instrumentation
+  c.ci.instrument :rspec
 end
 ```
 
-Run your tests as you normally do, specifying the environment where test are being run (for example, `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) in the `DD_ENV` environment variable. For example:
+Run your tests as you normally do, specifying the environment where test are being run in the `DD_ENV` environment variable.
+
+You could use the following environments:
+
+* `local` when running tests on a developer workstation
+* `ci` when running them on a CI provider
+
+For example:
 
 ```bash
-DD_ENV=ci bundle exec rake cucumber
+DD_ENV=ci bundle exec rake spec
 ```
 
 {{% /tab %}}
-{{% tab "RSpec" %}}
 
-The RSpec integration traces all executions of example groups and examples when using the `rspec` test framework.
+{{% tab "Minitest" %}}
 
-To activate your integration, add this to the `spec_helper.rb` file:
+The Minitest integration traces all executions of tests when using the `minitest` framework.
+
+To activate your integration, add this to the `test_helper.rb` file:
 
 ```ruby
-require 'rspec'
+require 'minitest'
 require 'datadog/ci'
 
+# Configure default Minitest integration
```

Comment:
nitpick: this comment is in minitest but not in the other frameworks. I don't know if it's intentional